### ISSUE TITLE
Different Values for saturated enthalpy Solkane and CoolProp

### ISSUE THREAD

**Issue by mboogi** [CoolProp/CoolProp#2088]
Hi to all!
was using Coolprop in Java, Kotlin and in Python lately, both on Windows and LInux (+Docker) altogether  with Micronaut and Spring and I have to say that after initial issues for compiling, it works very nice and fast! 

I am referencing my codes with help of solkane software, which means, when I program something I am checking the values in tables and diagrams but also in Solkane  refrigerant properties. 
Currently when i am trying to compare the values in superheated area, I am getting really high discrepancy:
for Refrigerant R134a, temperature 45 Degrees Celsius and 9 Bar, the value for enthalpy from Solkane is: 427.43 kJ/Kg, however when I calculate it in CoolProp, the value I am receiving is 442.0kJ/kg

Basically code I am using is just:

var enthalpy=CoolProp("H","T",318.15,"P|gas" 9*10000,"R134a")

I have tried also changing the reference state and the result is the same

Any chance if you could tell me why such high discrepancy occurred?
BR
MB

**CoolProp Version:** 6.4.2  
**Operating System and Version:** Windows 10, Ubuntu, Fedora, WSL  
**Access Method:** (Python, Java, Kotlin, Scala]

**Comment by ibell** [comment]
See our most popular faq:
https://github.com/CoolProp/CoolProp/blob/master/FAQ.md#usage

Setting the reference state should work though.   Refprop and Coolprop use
the same reference state for R134a.

**Comment by mboogi** [comment]
Hi Ian,
here is my pseudocode
```
CoolProp.set_reference_state('R134a','ASHRAE')

CoolProp.PropSI('H', 'T', 233.15, 'Q', 0, 'R134a')

Reference Point Enthalpy should be close to zero:  -1.054663405404959E-11

CoolProp.PropSI('H', 'T', 318.15, 'P|gas', 116000.0, 'R134a')

Enthalpy relative to Ashrae reference point: 293487.1640940012

CoolProp.PropSI('H', 'T', 233.15, 'Q', 0, 'R134a')

Enthalpy at Ashrae reference point: 148144.04579930683

Enthalpy at Ashrae reference Point: 148144.04579930683 ,enthalpy relative to reference point: 293487.1640940012 , sum = 441631.209893308

CoolProp.set_reference_state('R134a','DEF')

-------------------------------------------------------------------------------------------------------------------

 

CoolProp.PropSI('H', 'T', 233.15, 'P|gas', 116000.0, 'R134a')

Enthalpy: 441631.20989330806

```

The results is in both cases 441631 , which is understandable. 
However when I check out the Solkane software for the same conditions i am getting 421440J/Kg

do you know why is  there  such a discrepancy?

**Comment by ibell** [comment]
Maybe you should ask Solvane? Ask them what EOS they use.

**Comment by ibell** [comment]
Closing as part of an issue-tracker cleanup — this issue has had no activity in over two years.

*To be transparent: this comment and the close action are being performed by Claude (Anthropic's AI coding assistant), running on behalf of @ibell. Ian reviewed and approved the bulk-close criteria; I am the one executing the action.*

If you would like this issue reconsidered, please feel free to **re-open it** — a fresh comment with current behaviour against the latest CoolProp release is the most useful way to revive the discussion.

— Claude (acting for @ibell)
